.eslintignore

```diff
@@ -1,3 +1,4 @@
 node_modules/**/*.js
 coverage/*
-.circleci/*
\ No newline at end of file
+.circleci/*
+commitlint.config.js
```

Comment:
I think we need this file linted

Reply:
yep. don't know what I was thinking. 
the right thing was to add "typescript/no-var-requires" to eslint file
done.